src/AblaufSteuerung.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+
+namespace de.brick {
+
+    /**
+    Steuert den Ablauf des Algorithmus auf einer abstrakten Ebene.
+    */
+    public class AblaufSteuerung
+    {
+        private readonly Population population;
+
+        public AblaufSteuerung(Population population) {
+            this.population = population;
+        }
+
+        /**
+        @startuml
+        (*) --> Initialisierung
+        --> Fitness
+        if "Terminierung?" then
+            --> [weiter] Selektion
+            --> Kreuzung
+            --> Mutation
+            --> [nächste Generation] Fitness
+        else
+            --> [fertig] (*)
+        endif
+        @enduml        
+         */
+        public void starten() {
+            population.initialisieren();
+            population.fitnessTesten();
+            while (population.nichtTerminiert()) {
+                population.selektiereNeueGeneration();
+                population.kreuzeVorfahren();
+                population.mutiereChromosom();
```

Comment:
muss die "fitnessTesten()" nicht auch in der schleife aufgerufen werden?
das ergebnis von dem test ist doch eine der bedingungen, die in "nichtTerminiert()" einfließt.